gradlew

```diff
@@ -28,16 +44,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Why do we have this change?

Reply:
I don't know why gradle changed it in their template. I just run the normal upgrade command gradlew wrapper --gradle-version 6.9.1, nothing special